plugins/source/aws/resources/services/ram/resource_share_associated_principals_fetch.go

```diff
@@ -18,7 +18,6 @@ func fetchRamResourceShareAssociatedPrincipals(
 ) error {
 	input := &ram.GetResourceShareAssociationsInput{
 		AssociationType: types.ResourceShareAssociationTypePrincipal,
-		Principal:       parent.Item.(types.ResourceShare).ResourceShareArn,
```

Comment:
why do we skip this? in this case it's no longer a relation

Reply:
`ResourceShareArn` is not a a principal (accountID, orgid, user or role)....

Reply:
It is now a top level table